### PR TITLE
Add CNTK parser of MMdnn

### PR DESCRIPTION
**Pull request by kitstar**

### CODE REVIEW

**Comment by ysh329** [comment]
Great Again! 👍

File: README.md [hunk 1]
```diff
@@ -12,15 +12,15 @@ The sheet below is a overview of all convertors in github (not only contain offi
...
+|**[pytorch](https://github.com/pytorch/vision)** :sparkles:| None | [pytorch2caffe](https://github.com/longcw/pytorch2caffe) [pytorch-caffe-darknet-convert](https://github.com/marvis/pytorch-caffe-darknet-convert) | [onnx-caffe2](https://github.com/onnx/onnx-caffe2) | ONNX | None | None |   -   | None | [pytorch2keras](https://github.com/nerox8664/pytorch2keras) [nn-transfer](https://github.com/gzuidhof/nn-transfer) | [pytorch-caffe-darknet-convert](https://github.com/marvis/pytorch-caffe-darknet-convert) | [pytorch2keras](https://github.com/nerox8664/pytorch2keras) (over Keras) | None | [onnx-coreml](https://github.com/onnx/onnx-coreml) |
```
Comment (steadicat) [inline README.md:21]: Did you mean to remove `torch2coreml`?

Comment (kitstar) [reply README.md:21]: Yes. The torch2coreml is a convert from torch7->coreml. I thought it should not be placed in pytorch->coreml cell.

Comment (steadicat) [reply README.md:21]: Ah. You’re right. I was mislead by the `pytorch` dependency, and by [this thread](https://forums.developer.apple.com/thread/85525).

**Pull request merged**